### ISSUE TITLE
After build, clang++ is put into 'stac' instead of 'stack' 

### ISSUE THREAD

**Issue by msoos** [xiw/stack#1]
``` bash
~/development/llvm/stack/build/bin$ ls
ncpu  optck  poptck  stac  stack  stack-build
```

and:

``` bash
~/development/llvm/stack/build/bin/stac$ ls -lah
total 8.0K
drwxrwxr-x 2 soos soos 4.0K Sep 22 23:35 .
drwxrwxr-x 4 soos soos 4.0K Sep 22 23:43 ..
lrwxrwxrwx 1 soos soos   21 Sep 22 23:35 clang++ -> ../../../test/clang++
```

I believe this is a typo :)

**Comment by zeldovich** [comment]
Thanks, will fix.

**Comment by xiw** [comment]
fixed in 2cb60cb4, thanks.
